src/App.js

```diff
@@ -0,0 +1,131 @@
+import './App.css';
+import React, {useState} from 'react';
+import Counter from "./assets/Components/fruitCounter";
+
+
+function App() {
+    const [bananaCounter, setBananaCounter] = useState(0)
+    const [aambeienCounter, setAambeienCounter] = useState(0)
+    const [appelCounter, setAppelCounter] = useState(0)
+    const [kiwiCounter, setKiwiCounter] = useState(0)
+    const [voornaam, setVoornaam] = useState('')
+    const [achternaam, setAchternaam] = useState('')
+    const [leeftijd, setLeeftijd] = useState('')
+    const [postcode, setPostcode] = useState('')
+    const [bezorgFrequentie, setBezorgFrequentie] = useState('')
+    const [bezorgMoment, setBezorgMoment] = useState('overdag')
+    const [opmerking, setOpmerking] = useState('')
+    const [akkoord, setAkkoord] = useState(false)
```

Comment:
Misschien dat je in het vervolg iets met `react-hook-form` kan doen. Dat maakt het maken van een form een stuk eenvoudiger dan met de hoeveelheid useState's 😄